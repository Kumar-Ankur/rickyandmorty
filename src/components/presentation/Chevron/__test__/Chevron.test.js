import React from 'react';
import { shallow } from 'enzyme';
import Chevron from '../Chevron';

describe('check chevron component', () => {

    test('generate snapshot for chevron component', () => {
        const wrapper = shallow(<Chevron />);
        expect(wrapper).toMatchSnapshot();
    });

    test('check chevron component render without crash', () => {
        const wrapper = shallow(<Chevron />);
        const componentChevron = wrapper.find(`[data-test='component-chevron']`);
        expect(componentChevron.length).toBe(1);
    });
})