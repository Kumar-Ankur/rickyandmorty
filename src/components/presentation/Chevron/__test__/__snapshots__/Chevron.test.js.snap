// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check chevron component generate snapshot for chevron component 1`] = `ShallowWrapper {}`;
